### PR TITLE
[feature] manage dependencies version by parent pom

### PR DESCRIPTION
**Pull request by Aias00**
## What's changed?

manage dependencies version by parent pom

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [x]  I have written the necessary doc or comment.
- [x]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [x] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: push/pom.xml [hunk 1]
```diff
@@ -28,8 +28,6 @@
     <artifactId>hertzbeat-push</artifactId>
     <name>${project.artifactId}</name>
     <properties>
```
Comment (yuluo-yx) [inline push/pom.xml:30]: Remove the properties tag as well, right?

File: remoting/pom.xml [hunk 2]
```diff
@@ -28,8 +28,6 @@
     <artifactId>hertzbeat-remoting</artifactId>
     <name>${project.artifactId}</name>
     <properties>
```
Comment (yuluo-yx) [inline remoting/pom.xml:30]: ditto

File: warehouse/pom.xml [hunk 3]
```diff
@@ -24,13 +24,6 @@
         <version>2.0-SNAPSHOT</version>
     </parent>
     <properties>
```
Comment (yuluo-yx) [inline warehouse/pom.xml:26]: ditto

**Review by yuluo-yx — APPROVED** [review]
lgtm

**Review by tomsun28 — APPROVED** [review]

**Pull request merged**